bundles/org.openhab.binding.comfoair/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -833,6 +849,100 @@
 		<state readOnly="true"/>
 	</channel-type>
 
+	<channel-type id="is_preheater" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Preheater</label>
+		<description>Preheater option installed</description>
+		<category>Switch</category>
+		<state readOnly="false"/>
```

Comment:
The readme states, these are read only.

Reply:
I'm not sure if I understand your comment correctly, but as stated above these channels are NOT meant to be read only

Reply:
Please ignore my comment, was a bit to late yesterday.